erpnext/manufacturing/doctype/production_plan/production_plan.py

```diff
@@ -63,8 +65,13 @@ def get_pending_material_requests(self):
 		if self.item_code:
 			item_filter += " and mr_item.item_code = %(item)s"
 
+		if self.from_delivery_date:
+			mr_filter += " and mr.schedule_date >= %(from_delivery_date)s"
```

Comment:
Label should be Required date for the material request instead of delivery date, add separate fields for better understanding